File: Python-Task-05/C Files/log_analyzer.c
#include<stdio.h>
#include<string.h>

int main(){

char logs[6][10]={"Success","Failed","Failed","Success","Failed","Success"};

int success=0,failed=0;

for(int i=0;i<6;i++){

if(strcmp(logs[i],"Success")==0)
success++;
else
failed++;

}

printf("Total Attempts: 6\n");
printf("Successful Logins: %d\n",success);
printf("Failed Logins: %d\n",failed);

printf("\nMonitoring failed logins helps detect suspicious login attempts and improves cybersecurity.\n");

return 0;
}

File: Python-Task-05/Python Files/log_analyzer.py
logs = ["Success","Failed","Failed","Success","Failed","Success"]

success = logs.count("Success")
failed = logs.count("Failed")

print("Total Attempts:",len(logs))
print("Successful Logins:",success)
print("Failed Logins:",failed)

print("\nImportance:")
print("Monitoring failed logins helps detect brute-force attacks and unauthorized access attempts.")